examples/kotlin-react/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
You need to have your node_modules directory so that people can test the examples without having to download a ton of dependencies or even have npm installed. You need to include the basic dependencies that are essential for your app to work and pass the tests.
https://github.com/tastejs/todomvc/blob/master/app-spec.md#dependency-management

Reply:
https://travis-ci.org/tastejs/todomvc/jobs/402660648

Reply:
Ok, I forgot that the node_modules was in the .gitignore, will remove and add to the commit.. I also integrated to the cypress tests and are 3 main tests that are still failing, I am fixing these problems and uploading the fixed version as soon as possible. 

When the tests pass successfully I'll push the changes and let you know..

Thanks..